.github/workflows/e2e.yml

```diff
@@ -36,22 +36,25 @@ jobs:
       # resolution options, using npm release or a gh ref:
       #
       # option #1, set the version of stellar-sdk based on a npm release version
-      SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION: v11.0.0-beta.6
+      SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION: 11.0.1
       # option #2, set the version of stellar-sdk used as a ref to a gh repo if
       # a value is set on SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO, it takes
       # precedence over any SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION
-      SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO:
-      SYSTEM_TEST_JS_STELLAR_SDK_GH_REF:
+      SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO: 
+      SYSTEM_TEST_JS_STELLAR_SDK_GH_REF: 
+
+      # the version of rs-stellar-xdr to use for quickstart
+      SYSTEM_TEST_RS_XDR_GIT_REF: v20.0.1
 
       # system test will build quickstart image internally to use for running the service stack
       # configured in standalone network mode(core, rpc)
-      SYSTEM_TEST_QUICKSTART_GIT_REF: https://github.com/stellar/quickstart.git#69a1089eee9aaac4bafe4cbfc0639de18222db6a
+      SYSTEM_TEST_QUICKSTART_GIT_REF: https://github.com/stellar/quickstart.git#412bb828ddb4a93745227ab5ad97c623d43f3a5f
 
       # triggers system test to log out details from quickstart's logs and test steps
       SYSTEM_TEST_VERBOSE_OUTPUT: "true"
 
       # the soroban test cases will compile various contracts from the examples repo
-      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "main"
+      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "6160782e48cb30d31e1cd61f06cb8299370acf26"
```

Comment:
is this the right hash? looking at https://github.com/stellar/soroban-examples/commits/main/ we can update it to either c7947120dc3ef92345d6e019737065d916cfae9d or keep it at `main`, now

Reply:
good eye, yeah, at the time the hot fix branch was created that was the latest hash on main with updates to v20 sdk, but the repo wasn't tagged yet, so used what was available, but looks like [v20.0.0](https://github.com/stellar/soroban-examples/releases/tag/v20.0.0) was created, so, can pull that in here now, it's materially the same contract code.